.claude/skills/build-plugin/references/ui.md

```
@@ -226,18 +285,28 @@ Defines the config form shown when a user adds the plugin. One entry per config
 ```
 
-Add `"displayAs": "fieldGroupToggle"` to make the group collapsible:
-
-```json
-{ "type": "fieldGroup", "name": "advanced", "label": "Advanced Options", "displayAs": "fieldGroupToggle",
-  "fields": [ ...field definitions... ] }
-```
+**`displayAs`** controls how the group renders:
+
+- `"fieldGroupToggle"` — a collapsible/switchable group:
+    ```json
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)